site/data/qasida/hibbi-injala-jamaluh.json

```diff
@@ -0,0 +1,251 @@
+{
+    "name": {
+        "ara": "حِبّي انجَلَى جَمَالُه",
+        "en": "Hibbī Injalā Jamāluh - My beloved's beauty has been revealed"
+    },
+    "verses": {
+        "ara": [
+
+            ["حِبّي انجَلَى جَمَالُه"],
+            ["وقَد حان وصالُه"],
+            ["أرانيَ كمالُه"],
```

Comment:
I want to see if we can get all the harakaat for each line, that would be my preference (mainly to help people like me that can't figure out the right one to use most of the time!)